### ISSUE TITLE
[Bug]: milvus-standalone exited with code 2

### ISSUE THREAD

**Issue by miche2020** [milvus-io/milvus#16193]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.0.1
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus==2.0.0
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: N/a
- Others:
```

### Current Behavior

When i start docker compose with filled vectors volumes, I have an error milvus-standalone exited with code 2. When I delete volumes docker compose up starts correctly. 
[log.txt](https://github.com/milvus-io/milvus/files/8348247/log.txt)
[docker-compose.txt](https://github.com/milvus-io/milvus/files/8348260/docker-compose.txt)
Thanks!

### Steps To Reproduce

```markdown
1 docker compose up
2 fill vectors in milvus db
3 docker compose down
4 docker compose up
```

**Comment by yanliang567** [comment]
@miche2020 checking the logs, I found it actually failed when trying to load the previous filled vectors into memory. It looks like known issue in 2.0.1, which was fixed in master. Could you please retry with master build or just waiting for 2.0.2 release. there would be a 2.0.2 release early next week.

/assign @miche2020 
/unassign

**Comment by miche2020** [comment]
> @miche2020 checking the logs, I found it actually failed when trying to load the previous filled vectors into memory. It looks like known issue in 2.0.1, which was fixed in master. Could you please retry with master build or just waiting for 2.0.2 release. there would be a 2.0.2 release early next week.
> 
> /assign @miche2020 /unassign

Can i use milvus 2.0.0?

**Comment by yanliang567** [comment]
Mmm, if you don't mind the issues fixed in 2.0.1 and later.  But personally I suggest you use the latest one.
